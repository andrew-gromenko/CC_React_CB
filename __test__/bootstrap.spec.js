import React from 'react';
import {shallow, mount} from 'enzyme';
import renderer from 'react-test-renderer';
import Bootstrap from '../src/application/bootstrap';

describe('Bootstrap', () => {

	it('renders correctly', () => {
		const tree = renderer.create(
			<Bootstrap />
		).toJSON();

		expect(tree).toMatchSnapshot();
	});
});