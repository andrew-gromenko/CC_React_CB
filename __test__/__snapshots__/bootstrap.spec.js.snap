// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Bootstrap renders correctly 1`] = `
<div>
  <div
    className="container-fluid head"
  >
    <header
      className="boilerplate-heading"
    >
      <div
        className="row align-items-center"
      >
        <div
          className="col-md-3"
        >
          <figure
            className="logo-image"
          >
            <img
              alt="Eggcllnt greeting"
              src="/assets/images/eggcllnt-head.gif"
            />
          </figure>
        </div>
        <div
          className="col-md-9"
        >
          <h1
            className="align-self-center"
          >
            Hello, world
          </h1>
          <p>
            Make things better with React
          </p>
        </div>
      </div>
    </header>
  </div>
</div>
`;
